cmd/skopeo/sync.go

```diff
@@ -104,6 +105,7 @@ See skopeo-sync(1) for details.
 	flags.StringVarP(&opts.destination, "dest", "d", "", "DESTINATION transport type")
 	flags.BoolVar(&opts.scoped, "scoped", false, "Images at DESTINATION are prefix using the full source image path as scope")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Copy all images if SOURCE-IMAGE is a list")
+	flags.BoolVarP(&opts.continueOnError, "ignore", "", false, "Abort the sync if any image ref sync failed")
```

Comment:
(Absolutely non-blocking: _After_ the option name settles, consider whether  the `continueOnError` field should be named similarly to the option. OTOH `continueOnError` is a pretty great internal name, it might be better than a short-hand external option name.)

Reply:
Now that the PR uses --keep-going, I think calling that internal variable `keepGoing` instead of `continueOnError` is slightly better. So I've updated the PR to rename this internal member.